src/App.tsx

```diff
@@ -1,24 +1,16 @@
 import React, { FC } from 'react';
-import logo from './logo.svg';
-import './App.css';
 
```

Comment:
I suggest removing this line.

Reply:
Resolved on edaae768852f37c715486f98be7aaa9cecb81444